src/OpenTelemetry.Exporter.InMemory/InMemoryExporterHelperExtensions.cs

```diff
@@ -42,6 +42,14 @@ public static TracerProviderBuilder AddInMemoryExporter(this TracerProviderBuild
                 throw new ArgumentNullException(nameof(exportedItems));
             }
 
+            if (builder is IDeferredTracerProviderBuilder deferredTracerProviderBuilder)
+            {
+                return deferredTracerProviderBuilder.Configure((sp, builder) =>
+                {
+                    builder.AddProcessor(new SimpleActivityExportProcessor(new InMemoryExporter<Activity>(exportedItems)));
```

Comment:
Code looks fine but there's really no reason to do this because InMemoryExporter doesn't have any options.

Reply:
This helps with the ordering being consistent with other exporters, when using Hosting packge.

Reply:
This is being done to ensure correct ordering of processing. If a custom processor is setup as mentioned in #2126, then the `InMemoryExporter` will not receive the `Activity` processed by the custom processor.

Reply:
Got it! I don't know if the issue is still valid because we removed the `AddProcessor<T>` extension but no harm in keeping it consistent.